.github/workflows/website_tests.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: Website Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DEBIAN_FRONTEND: noninteractive
+      LANG: en_US.UTF-8
+      LANGUAGE: en_US:en
+      LC_ALL: en_US.UTF-8
+      PODEXEC: 'podman exec --workdir /tmp/website --env-file .env ovirt-web-tests'
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v2
+
+      - name: "Cache files"
+        uses: actions/cache@v2
+        id: website-cache
+        with:
+          path: tmp/.htmlproofer
+          key: htmlproofer
+
+      - name: "Check for obsolete wiki headers"
+        run: |
+          if grep -rEi "^wiki_[^ ]+: " source/; then
+            echo '/!\ Wiki headers are obsolete (see previous log entries)'
+            exit 2
+          fi
+
+      - name: "Check for nonbreaking space in markdowns"
+        run: |
+          if grep -P '\xa0' `find source/ -name "*.md"` ; then
+            echo '/!\ nonbreaking spaces have been found within the markdown (see previous log entries)'
+            exit 2
+          fi
+
+      - name: "Install Podman"
+        run: sudo apt-get install -y podman
+
+      - name: "Create the Website Build Environment container"
+        # Duck: DO NOT change the target OS, it needs to be in sync with the production builder
+        run: podman run -dt --name ovirt-web-tests --volume .:/tmp/website registry.fedoraproject.org/fedora:30
```

Comment:
Fedora 30 reached EOL on 2020-05-26 ; any chances that we can move to more recent Fedora?

Reply:
Yes I wanted to upgrade too but did not get around to do it. Now we can bump the version in a PR and see if it would work before reinstalling the builder.